src/components/SavedFilters.vue

```diff
@@ -52,7 +52,15 @@
   }))
 
   const router = useRouter()
-  const { value: myCustomDefaultFilter } = useCustomDefaultFlowRunsFilter()
+  const { value: mySavedCustomDefaultFilter, asFlowRunsFilter: myCustomDefaultFilter } = useCustomDefaultFlowRunsFilter()
+  const nameOfDefaultFilter = computed(() => {
+    const customDefault = mySavedCustomDefaultFilter.value
+    if (!savedSearches.value.length || !customDefault) {
+      return systemDefaultSavedSearch.name
+    }
+    // TODO: handle case where customDefault is not in savedSearches
```

Comment:
I'm thinking of prompting a user that their saved default filter might not exist anymore - [clear it] or [save it as a new filter]. 

Need to think through where/how to do that w/o just introducing a side effect into this computed ref.

Reply:
eh. popping a toast after deleting from localstorage for now.

Reply:
Reworked this to instead surface this as "Unsaved (default)" which is only visible locally to that user until they change their default _or_ save the unsaved (default) with a name.